samples/AspNetCore/SecretStoreConfigurationProviderSample/SecretStoreConfigurationProviderSample.csproj

```diff
@@ -0,0 +1,12 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
```

Comment:
Could you please add a Redme.md as well.

Reply:
> Could you please add a Redme.md as well.

just added /docs/get-started-dapr-secret-store-configuration-provider-aspnet-core.md and squashed...